app/renderer/bookmarks.js

```diff
@@ -0,0 +1,323 @@
+/*
+* Vieb - Vim Inspired Electron Browser
+* Copyright (C) 20
```

Comment:
This seems incorrect, could you remove the extra newlines and make sure to update the year if you update the file? There are plugins for most editors to do this automatically too, here's one for Vim: https://github.com/antoyo/vim-licenses